client/components/learn/Resources/resources-model.ts

```diff
@@ -41,6 +41,11 @@ export const RESOURCES: Record<ResourceCategory, ResourceDescription[]> = {
       url: 'https://validator.w3.org/',
       cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/w3c-validator_500w.png`,
     },
+    {
+      title: 'PX <-> Rem Converter',
```

Comment:
Let's drop this tool, if you understand what `rem` is all about then this calculation is just a division.